### PR TITLE
Set telemetry-group for the chaos configs

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This will ease search and download of data for product under test via telemetry service.

### CODE REVIEW

File: ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__4.14-nightly-arm.yaml [hunk 1]
```diff
@@ -36,9 +36,7 @@ tests:
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
     env:
-      BASE_DOMAIN: aws.rhperfscale.org
-      OCP_ARCH: arm64
```
Comment (paigerube14) [inline ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__4.14-nightly-arm.yaml:40]: do you want to remove this?

**Comment by chaitanyaenr** [comment]
/pj-rehearse pull-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-415-hog

**Comment by chaitanyaenr** [comment]
/pj-rehearse pull-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-415-hog

**Comment by chaitanyaenr** [comment]
/pj-rehearse pull-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-415-hog

**Review by tsebastiani — APPROVED** [review]

**Pull request merged**